src/pages/_app.tsx

```diff
@@ -78,77 +86,40 @@ export default class App extends NextApp<{ authUserData: LoggedUser; }> {
   }
 
   render() {
-    // @ts-ignore
-    const { Component, pageProps, me, userAgent, authUserData } = this.props
+    // @ts-expect-error
+    const { Component, pageProps, userAgent } = this.props
 
-    const SSRDataValue = { me, userAgent }
+    const SSRDataValue = { userAgent }
 
-    if (!this.store) {
-      const { store, persistor } = bootstrapStore()
-      this.store = store
-      this.persistor = persistor
+    if (!isSSR) {
+      CookiesAuthUserTokenStorage.initToken().then()
```

Comment:
Je crois qu'on avait vu qu'il n'y avait pas besoin du then
```suggestion
      CookiesAuthUserTokenStorage.initToken()
```